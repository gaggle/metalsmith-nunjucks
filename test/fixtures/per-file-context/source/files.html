{% files_keys_tag %}
