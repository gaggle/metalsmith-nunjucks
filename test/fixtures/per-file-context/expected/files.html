file.html,filename.html,files.html
